File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.cpp
class Solution {
public:
    int maxFreqSum(string s) {
        int freq[26] = {0};
        for(char c: s){
            freq[c-'a'] += 1;
        }
        int vowels[5] = {0,4,8,14,20};
        int maxVowel = 0;
        for (int i : vowels){
            if (freq[i] > maxVowel){
                maxVowel = freq[i];
            }
            freq[i] = 0;
        }
        int maxCons = *max_element(freq,freq+26);
        return maxVowel + maxCons;
    }
};

File: 3872-find-most-frequent-vowel-and-consonant/find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        freq = [0]*26
        for c in s:
            freq[ord(c)-ord('a')] += 1
        maxVowelFreq = 0
        for i in (0,4,8,14,20):
            if freq[i] > maxVowelFreq:
                maxVowelFreq = freq[i]
            freq[i] = 0
        maxConsFreq = max(freq)
        return maxVowelFreq + maxConsFreq